nikola/data/themes/bootblog4-jinja/templates/base_helper.tmpl

```diff
@@ -154,26 +155,7 @@ lang="{{ lang }}">
 
 
 {% macro html_feedlinks() %}
-    {% if rss_link %}
-        {{ rss_link }}
-    {% elif generate_rss %}
-        {% if translations|length > 1 %}
-            {% for language in translations|sort %}
-                <link rel="alternate" type="application/rss+xml" title="RSS ({{ language }})" href="{{ _link('rss', None, language) }}">
-            {% endfor %}
-        {% else %}
-            <link rel="alternate" type="application/rss+xml" title="RSS" href="{{ _link('rss', None) }}">
-        {% endif %}
-    {% endif %}
-    {% if generate_atom %}
-        {% if translations|length > 1 %}
-            {% for language in translations|sort %}
-                <link rel="alternate" type="application/atom+xml" title="Atom ({{ language }})" href="{{ _link('index_atom', None, language) }}">
-            {% endfor %}
-        {% else %}
-            <link rel="alternate" type="application/atom+xml" title="Atom" href="{{ _link('index_atom', None) }}">
-        {% endif %}
-    {% endif %}
+    {{ feeds_translations.head(classification=None, kind='index', other=False) }}
```

Comment:
The generated HTML will be different than the one currently there, but I don't think that's a bad thing :-)

(Also it will make changes a lot easier in the future...)